imports/ui/pages/professor/questions_public.jsx

```diff
@@ -79,21 +79,41 @@ class _QuestionsPublic extends Component {
     if (!_.findWhere(newQuestions, {_id: this.state.selected})) {
       this.setState({ selected: null, questions: newQuestions, questionMap: _(newQuestions).indexBy('_id') })
     } else this.setState({questions: newQuestions, questionMap: _(newQuestions).indexBy('_id')})
+    
   }
 
   limitAndUpdate (data) {
     this.setState({limit: 11}, () => this.updateQuery(data))
   }
 
   componentWillReceiveProps () {
-    const newQuestions = Questions.find(this.state.query.query, this.state.query.options).fetch()
+    const user = Meteor.user()
```

Comment:
Presumably this does not need to be changed if we have the publication changed (see other comment)